pkg/check/settlements/settlements.go

```diff
@@ -148,7 +152,7 @@ func Check(c *bee.Cluster, o Options, pusher *push.Pusher, pushMetrics bool) (er
 func DryRunCheck(c *bee.Cluster, o Options) (err error) {
 	ctx := context.Background()
 
-	ng := c.NodeGroup(o.NodeGroup)
+	ng := c.NodeGroup("skip")
```

Comment:
here too. also, will `NodeGroup` fail for a non-existing node group? panic?

Reply:
It will return nil, but I have fixed this tests to use cluster level, the same as with other tests.